code-for-canada/project-thundercat
# [Vulnerability Patch] Fixed 2 high and 1 medium vulnerabilities

# Description

This patch resolved the following vulnerabilities:
- [js-yaml](https://app.snyk.io/test/npm/js-yaml) (2)
- [handlebars](https://app.snyk.io/test/npm/handlebars)

Upgrades:
- "mem": ">=4.1.0" to "mem": ">=4.3.0" (https://snyk.io/vuln/npm:mem:20180117)
- "react-script": "2.1.5" to "react-script": "2.1.8" (https://app.snyk.io/test/npm/react-scripts/)

## Type of change

- Dependency change

## Screenshot

**Before**
---

**js-yaml (HIGH)**
![image](https://user-images.githubusercontent.com/23021242/56209941-c1515c00-6022-11e9-8a90-7e2c6dfe6e4e.png)
More about this issue: https://app.snyk.io/vuln/SNYK-JS-JSYAML-174129 

**handlebars (HIGH)**
![image](https://user-images.githubusercontent.com/23021242/56210020-ee057380-6022-11e9-84e8-bd87eceac382.png)
More about this issue: https://app.snyk.io/vuln/SNYK-JS-HANDLEBARS-174183

**js-yaml (MEDIUM)**
![image](https://user-images.githubusercontent.com/23021242/56210068-037a9d80-6023-11e9-8622-9779d4aad7ad.png)
More about this issue: https://app.snyk.io/vuln/SNYK-JS-JSYAML-173999

**After**
---
![image](https://user-images.githubusercontent.com/23021242/56210175-3c1a7700-6023-11e9-8daa-c38f6c8ff659.png)


# Testing

Manual steps to reproduce this functionality:

1.  Delete frontend/node_modules folder
2.  Run docker-compose up
3.  Verify that application works as expected

# Checklist

Applicable for all code changes.
